### ISSUE TITLE
SIGNATURE 1 TEST shows "MODIFIED" for some NSPs

### ISSUE THREAD

**Issue by WanderMax** [julesontheroad/NSC_BUILDER#134]
For some NSPs, if I run verify step, it shows **is modified** in SIGNATURE 1 TEST, but this NSP runs well in switch console.
Here is the snapshot as below:

![image](https://user-images.githubusercontent.com/15909056/85857919-4f130480-b7ed-11ea-9f18-19a77aee2a3d.png)

Is this NSP good, or why?
Thanks.

**Comment by julesontheroad** [comment]
Probably converted with 4nxci or something similar, any method that patches signature2 modifies the region hashed by signature1, signature 1 is made with Nintendo public key and verifiable with it's public pair. Since only Nintendo can make a valid signature anything that verifies fine against it can be deemed as legit.
NSCB also verifies files that only modify things in the header or some other places where the changes can be predicted recalculating the original state of the file. Basically it recalculate the original header and verifies it against the signature 1. These files are also restorable with the "Restore Mode".
So basically if the decription test is correct you have a working nsp, if the signature 1 test is incorrect you have a file that could be a patched file in a way is not restorable or a bricker. Chances are is just a convert since control and manual nca are legit but **show the isgamecard was changed** so i'd say is just a file converted with 4nxci or a similar tool.
